test/xchar-test.cc

```diff
@@ -315,10 +315,12 @@ TEST(xchar_test, ostream) {
 #endif
 }
 
+#ifdef FMT_XCHAR_TEST_ENABLE_FORMAT_MAP
```

Comment:
Why did you add this condition?

Reply:
i tried to explain that above. `is_formattable<std::wstring>` is false locally, so it looked to me that `is_formattable` is also broken for `std::wstring` (and `std::map`), and i just wanted to fix it for `std::tuple`.

Reply:
well... maybe not `std::map`, but at least it seemed to give a false negative for `std::wstring`.

Reply:
i might be missing something basic though. i thought it should be `true` if `std::format` worked for it.

Reply:
`is_formattable` takes the char type as its [second template parameter](https://github.com/fmtlib/fmt/blob/8e47cfd1cdfa9cb29cbb5ae037124cd71425a1fe/include/fmt/core.h#L1800-L1801), which `is_tuple_formattable` isn't "acknowledging".

Reply:
ah! thanks! fixed.

Reply:
I think, this are bug, because this is copy of range test:
https://github.com/fmtlib/fmt/blob/eaa8efb950be5d8f1803a99b06f76cf398c67cb8/test/ranges-test.cc#L58-L61

Reply:
@phprus ya, i didn't take the `Char` parameter into account. i fixed the bug and removed the condition.